youtube_dl/extractor/bandcamp.py

```diff
@@ -415,3 +408,55 @@ def _real_extract(self, url):
             'episode_id': compat_str(video_id),
             'formats': formats
         }
+
+
+class BandcampUserIE(InfoExtractor):
+    IE_NAME = 'Bandcamp:user'
+    _VALID_URL = r'https?://(?:(?P<id>[^.]+)\.)?bandcamp\.com/?'
+
+    _TESTS = [{
+        'url': 'https://adrianvonziegler.bandcamp.com',
+        'info_dict': {
+            'id': 'adrianvonziegler',
+            'title': 'Albums of adrianvonziegler',
+        },
+        'playlist_mincount': 20,
+    }, {
+        'url': 'http://dotscale.bandcamp.com',
+        'info_dict': {
+            'id': 'dotscale',
+            'title': 'Albums of dotscale',
+        },
+        'playlist_count': 1,
+    }]
+
+    @classmethod
+    def suitable(cls, url):
+        return (False
+                if BandcampAlbumIE.suitable(url) or BandcampIE.suitable(url) or
+                BandcampWeeklyIE.suitable(url)
+                else super(BandcampUserIE, cls).suitable(url))
+
+    def _real_extract(self, url):
+        uploader = self._match_id(url)
+
+        webpage = self._download_webpage(url, uploader)
+
+        album_elements = re.findall(r'<a href="/(album/.[^"]+)">', webpage)
+
+        entries = [
+            self.url_result(
+                compat_urlparse.urljoin(url, album_id),
+                ie=BandcampAlbumIE.ie_key(),
+                video_id='%s-%s' % (uploader, album_id),
```

Comment:
Not a video id.

Reply:
I know, but I have no other way of specifying some kind of id from the information available in the user webpage.

I've removed the field.